README.md

```diff
@@ -124,7 +124,7 @@ npm install prosemirror-utils
    ```
 
 
- * **`findDomRefAtPos`**`(position: number, domAtPos: fn(pos: number) → {node: dom.Node, offset: number}) → dom.Node`\
+ * **`findDomRefAtPos`**`(position: number, domAtPos: fn(pos: number) → {node: dom.Node, offset: number}) → ?dom.Node`\
```

Comment:
this won't need to be optional if we opt to return the text/leaf node when childNodes is undefined :)